Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -39,12 +39,15 @@ enum AppSettingsDeeplinkOption {
 /// Child settings pages action
 protocol AppSettingsDelegate: AnyObject {
     func clickedVersion()
+    func clickedShowTour()
```

Comment:
nit: clicked is an odd choice of verb on a touch screen app but only a nit because the meaning is still clear.

Reply:
Oh, right. Probably better something like `pressedShowTour` ?